### ISSUE TITLE
nuScenes Validation Subset in Repo — Is It Example or Full Training Data?

### ISSUE THREAD

**Issue by jadeblank** [ika-rwth-aachen/MultiCorrupt#19]
Thank you for sharing your code and data. I am trying to reproduce your work and found the provided dataset helpful.

I noticed that the dataset you uploading in HuggingFace appears to be a  validation set of the nuScenes and the code in converter is just used to process nuscenes_infos_val.pkl. Could you kindly clarify if:

1、The model in the paper was trained and evaluated only on the validation set, or

2、This subset is provided as an example, and the full dataset (train/val/test) should be used for reproduction?

**Comment by TillBeemelmanns** [comment]
Hi,
thanks for your interest!

1. The models in the paper were trained on the train set of the nuScenes dataset and we followed strictly the training protocol of each proposed method. In some cases we did not train the model ourselves, but we took directly their uploaded weights. Hence, the models are not fine tuned or trained on the corrupted dataset.
2. In this work we only corrupt the val set of the nuScenes dataset and if you want to reproduce the results of this work you would need to evaluate the models on the corrupted validation sets. The uploaded dataset on huggingface contains the full corrupted validation set to reproduce the results of this work.  


Best,
Till

**Comment by jadeblank** [comment]
Hi,
Thank you for your detailed answer!  Your explanation is very clear, I have understood and the problem has been resolved now, thank you for your help!
